package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lderp-edir",
   "description": "Novell eDirectory extension for lderp",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
It shouldn't be - I intentionally continued to support the "helpful" attribute names.  Though I _did_ fix the given/surname mix-up.